pkg/controllers/metrics/pod/controller.go

```diff
@@ -84,8 +84,8 @@ type Controller struct {
 }
 
 func init() {
-	crmetrics.Registry.MustRegister(podGaugeVec)
-	crmetrics.Registry.MustRegister(podStartupTimeSummary)
+	crmetrics.Registry.MustRegister(podState)
```

Comment:
nit: Do we need these as duplicate MustRegister statements or can these be combined into a single one?